src/types/styles.ts

```diff
@@ -35,6 +35,11 @@ export enum Slic3rInfillStyle {
   Lightning = 5,
 }
 
+export enum SupportStylesCanvas {
+  grid = 0,
+  snug = 1,
+}
```

Comment:
Prefer singular naming for enum and capitalized member names:

```suggestion
export enum CanvasSupportStyle {
  Grid = 0,
  Snug = 1,
}
```